<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterDesignViewWidget</class>
 <widget class="QWidget" name="FilterDesignViewWidget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filter design</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout_4">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="1" column="0">
      <widget class="QGroupBox" name="m_groupBox_filterSettings_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Apply filter</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="m_label_applyFilterTo">
          <property name="text">
           <string>Apply filter to:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="m_comboBox_filterApplyTo">
          <item>
           <property name="text">
            <string>All</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MEG</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>EEG</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ECG</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>EOG</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>EMG</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QTabWidget" name="m_tabWidget_mode">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="Design">
           <attribute name="title">
            <string>Design Filter</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <layout class="QGridLayout" name="m_layout_designFilter">
              <item row="3" column="0">
               <widget class="QLabel" name="m_label_transitionBand">
                <property name="text">
                 <string>Transition band (Hz):</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="m_label_lowpass">
                <property name="text">
                 <string>From:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="m_label_designMethod">
                <property name="text">
                 <string>Design method:</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QSpinBox" name="m_spinBox_filterTaps">
                <property name="minimum">
                 <number>16</number>
                </property>
                <property name="maximum">
                 <number>4096</number>
                </property>
                <property name="singleStep">
                 <number>2</number>
                </property>
                <property name="value">
                 <number>4096</number>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QDoubleSpinBox" name="m_doubleSpinBox_to">
                <property name="suffix">
                 <string>Hz</string>
                </property>
                <property name="decimals">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <double>999.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
                <property name="value">
                 <double>40.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="m_label_filterTaps">
                <property name="text">
                 <string>Filter taps:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="m_label_highpass">
                <property name="text">
                 <string>To:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="m_comboBox_designMethod"/>
              </item>
              <item row="1" column="1">
               <widget class="QDoubleSpinBox" name="m_doubleSpinBox_from">
                <property name="suffix">
                 <string>Hz</string>
                </property>
                <property name="decimals">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <double>999.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
                <property name="value">
                 <double>1.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QDoubleSpinBox" name="m_doubleSpinBox_transitionband">
                <property name="suffix">
                 <string>Hz</string>
                </property>
                <property name="decimals">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <double>999.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
                <property name="value">
                 <double>0.100000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="4" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1" rowspan="5">
      <widget class="QGroupBox" name="m_groupBox_filterPlot">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Filter plot</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGraphicsView" name="m_graphicsView_filterPlot">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>450</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0" rowspan="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Load / Save</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="m_pushButton_exportFilter">
          <property name="text">
           <string>Save filter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_pushButton_loadFilter">
          <property name="text">
           <string>Load filter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_pushButton_exportPlot">
          <property name="text">
           <string>Export filter plot</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_comboBox_filterApplyTo</tabstop>
  <tabstop>m_tabWidget_mode</tabstop>
  <tabstop>m_pushButton_exportFilter</tabstop>
  <tabstop>m_pushButton_loadFilter</tabstop>
  <tabstop>m_pushButton_exportPlot</tabstop>
  <tabstop>m_graphicsView_filterPlot</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
